### ISSUE TITLE
Source mapping is wrong when you comment out script

### ISSUE THREAD

**Issue by dummdidumm** [sveltejs/language-tools#109]
**Describe the bug**
Source mapping is wrong when you comment out `script`.

**To Reproduce**

Add `"checkJs": true` to `jsconfig.json` / `tsconfig.json`

```html
<!-- <script>
const bla = true;

</script> -->

<Navigation />
```
Will do red ("cannot find Navigation") squiggels at `script`, not at `Navigation`.

**Expected behavior**
Everything should work the same as if script is not commented out

**System (please complete the following information):**
 - OS: Windows
 - IDE: VSCode
 - Plugin/Package: Svelte Beta

**Additional context**
Maybe that's something inside `svelte2tsx`, not honoring the comment and thinking there is a `script` tag.

**Comment by dummdidumm** [comment]
No longer reproducible, maybe fixed through #251

**Comment by dummdidumm** [comment]
No longer reproducible.
